test/language/expressions/object/method-definition/early-errors-object-method-arguments-in-formal-parameters.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016 Microsoft, Inc. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Brian Terlson <brian.terlson@microsoft.com>
+esid: pending
+description: It is a SyntaxError if FormalParameters contains arguments
+negative: SyntaxError
```

Comment:
This needs to be `onlyStrict` like early-errors-arrow-arguments-in-formal-parameters.js --- this restriction is only enforced in strict mode